docs/asciidoc/modules/ROOT/examples/generated-documentation/documentation.csv

```diff
@@ -285,12 +285,17 @@
+¦procedure¦apoc.trigger.install¦apoc.trigger.install(databaseName :: STRING?, name :: STRING?, kernelTransaction :: STRING?, selector :: MAP?, config = {} :: MAP?) :: (name :: STRING?, query :: STRING?, selector :: MAP?, params :: MAP?, installed :: BOOLEAN?, paused :: BOOLEAN?)¦CALL apoc.trigger.install(databaseName, name, statement, selector, config) | add a trigger kernelTransaction under a name, in the kernelTransaction you can use $createdNodes, $deletedNodes etc., the selector is {phase:'before/after/rollback/afterAsync'} returns previous and new trigger information. Takes in an optional configuration.¦true¦xref::background-operations/triggers.adoc
 ¦procedure¦apoc.trigger.list¦apoc.trigger.list() :: (name :: STRING?, query :: STRING?, selector :: MAP?, params :: MAP?, installed :: BOOLEAN?, paused :: BOOLEAN?)¦list all installed triggers¦true¦xref::background-operations/triggers.adoc
 ¦procedure¦apoc.trigger.pause¦apoc.trigger.pause(name :: STRING?) :: (name :: STRING?, query :: STRING?, selector :: MAP?, params :: MAP?, installed :: BOOLEAN?, paused :: BOOLEAN?)¦CALL apoc.trigger.pause(name) | it pauses the trigger¦true¦xref::background-operations/triggers.adoc
 ¦procedure¦apoc.trigger.remove¦apoc.trigger.remove(name :: STRING?) :: (name :: STRING?, query :: STRING?, selector :: MAP?, params :: MAP?, installed :: BOOLEAN?, paused :: BOOLEAN?)¦remove previously added trigger, returns trigger information¦true¦xref::background-operations/triggers.adoc
 ¦procedure¦apoc.trigger.removeAll¦apoc.trigger.removeAll() :: (name :: STRING?, query :: STRING?, selector :: MAP?, params :: MAP?, installed :: BOOLEAN?, paused :: BOOLEAN?)¦removes all previously added trigger, returns trigger information¦true¦xref::background-operations/triggers.adoc
 ¦procedure¦apoc.trigger.resume¦apoc.trigger.resume(name :: STRING?) :: (name :: STRING?, query :: STRING?, selector :: MAP?, params :: MAP?, installed :: BOOLEAN?, paused :: BOOLEAN?)¦CALL apoc.trigger.resume(name) | it resumes the paused trigger¦true¦xref::background-operations/triggers.adoc
+¦procedure¦apoc.trigger.start¦apoc.trigger.start(databaseName :: STRING?, name :: STRING?) :: (name :: STRING?, query :: STRING?, selector :: MAP?, params :: MAP?, installed :: BOOLEAN?, paused :: BOOLEAN?)¦CALL apoc.trigger.start(databaseName, name) | it resumes the paused trigger¦true¦xref::background-operations/triggers.adoc
+¦procedure¦apoc.trigger.stop¦apoc.trigger.stop(databaseName :: STRING?, name :: STRING?) :: (name :: STRING?, query :: STRING?, selector :: MAP?, params :: MAP?, installed :: BOOLEAN?, paused :: BOOLEAN?)¦CALL apoc.trigger.stop(databaseName, name) | it pauses the trigger¦true¦xref::background-operations/triggers.adoc
```

Comment:
I would like to propose that we add the word "eventually" to each description if you agree?